stdlib/xml/dom/minidom.pyi

```diff
@@ -87,31 +87,31 @@ class NamedNodeMap:
     def keys(self): ...
     def keysNS(self): ...
     def values(self): ...
-    def get(self, name, value: Any | None = ...): ...
+    def get(self, name: str, value: Any | None = ...): ...
     def __len__(self) -> int: ...
-    def __eq__(self, other: object) -> bool: ...
-    def __ge__(self, other: Any) -> bool: ...
-    def __gt__(self, other: Any) -> bool: ...
-    def __le__(self, other: Any) -> bool: ...
-    def __lt__(self, other: Any) -> bool: ...
-    def __getitem__(self, attname_or_tuple): ...
-    def __setitem__(self, attname, value) -> None: ...
-    def getNamedItem(self, name): ...
-    def getNamedItemNS(self, namespaceURI: str, localName): ...
-    def removeNamedItem(self, name): ...
-    def removeNamedItemNS(self, namespaceURI: str, localName): ...
-    def setNamedItem(self, node): ...
-    def setNamedItemNS(self, node): ...
-    def __delitem__(self, attname_or_tuple) -> None: ...
+    def __eq__(self, other: NamedNodeMap) -> bool: ...
```

Comment:
`__eq__` methods are a bit special: while it doesn't really make sense for a user to do a test like `if NamedNodeMap('foo', 'bar', 'baz') < object()` (so we'll be doing them a favour by being more restrictive in the type hints than the implementation is), it's perfectly fine for somebody to do a test like `if NamedNodeMap('foo', 'bar', 'baz') == object()`. This one should probably stay as it is:

```suggestion
    def __eq__(self, other: object) -> bool: ...
```